### ISSUE TITLE
Enhancement: desktop editor--js source coming from external file, autoloading

### ISSUE THREAD

**Issue by itsayellow** [wavedrom/wavedrom.github.io#12]
This is definitely lower-priority than functional fixes, but it would be super useful for me:

I love to edit my waveform .js code in an external text editor.  I then drag the .js file into the wavedrom source window to load my code to see how I'm doing.  This gets a little time-consuming on updates, where I have to delete the current code in wavedrom and re-drag the file into the window with my mouse each time I want to view updates.

It would be great if there was a possibility in the stand-alone editor to point to a particular file, and even better if there was some sort of reload button or auto-reloading that would happen when the .js file changed.

**Comment by drom** [comment]
@itsayellow good idea. It is possible, but we have to handle the case when edits made in both places. Or we can run `WaveDromEditor` in viewer mode?

**Comment by itsayellow** [comment]
I agree it would be simplest for it to be in view-only mode.  Then you wouldn't have to handle file saving from WaveDromEditor and keeping track of two sources of changes.  

For my purposes, I would be very happy with even a viewer mode only.  Depending on long-term plans for wavedrom, the other capabilities could always be added later.
